editor/audio/editor_audio_buses.cpp

```diff
@@ -1143,6 +1149,17 @@ void EditorAudioBuses::_notification(int p_what) {
 				save_timer->start();
 			}
 		} break;
+		case NOTIFICATION_VISIBILITY_CHANGED: {
+			if (is_visible()) {
+				_update_file_label_size();
+				if (use_default_editor_size) {
```

Comment:
This doesn't seem to work if Audio is the panel opened when editor starts. Though it's not important I suppose.
You could add a comment explaining what it does though.

Reply:
Yes, that's how it works.